ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -60,17 +62,27 @@
   @Autowired private MockMvc mvc;
   @Autowired private RestTemplate restTemplate;
 
-  private MockRestServiceServer server;
+  @Autowired
+  @Qualifier("nonBufferingRestTemplate")
+  private RestTemplate nonBufferingRestTemplate;
+
+  private MockRestServiceServer storeServer;
+  private MockRestServiceServer transformServer;
 
   @Before
   public void beforeEach() {
-    server = MockRestServiceServer.createServer(restTemplate);
+    MockRestServiceServerBuilder builder = MockRestServiceServer.bindTo(nonBufferingRestTemplate);
```

Comment:
Client and server need to use the same bean for MockServer to work. MockMvc intercepts the HTTP request send to the RestTemplate and redirects it to the mock server.